### PR TITLE
placeholder text

### LINKED ISSUE #299: Bug: Codeword and application status doesn't load right away
**Describe the bug**
On the application instructions page, the codeword and application status doesn't load right away. This is because TFH Dashboard is hosted on Heroku and it goes to sleep if it hasn't been accessed recently. This means that the page can't display the codeword and application status until TFH Dashboard wakes up and the main site is able to receive a response from the API. This could prevent people from knowing the codeword if it doesn't load by the time they finish reading the page.

**To Reproduce**
Clear your cache and visit the applications page at a time when it hasn't been accessed recently. Observe that the codeword and application status doesn't load.

**Expected behavior**
Some kind of loading screen or bar should be added to indicate that the codeword and application status is still loading, and potentially prevent people from viewing the application instructions until the page can fully load.

**Browser Information**
N/A

**Additional context**
If someone wants to take a crack at implementing this, please write the solution you are thinking of as a reply before submitting a PR.

### PR DESCRIPTION
**Pull request by leahcimto**
### Resolves:
Partially fixes #299

### Changes:

Adds placeholder text

Not sure if this is what you all meant by placeholder text, or you wanted to hide the whole sentence and put placeholder text there and then show the whole sentence?

### Local Tests:
None

### CODE REVIEW

File: forumhelpers/index.html [hunk 1]
```diff
@@ -49,9 +49,9 @@ <h3><a href="#curators" class="FHULink">Curators</a></h3>
 		</div>
 		<div class="forumHelpersList" id="forumHelpersList">
 			<h1 class="forumHelpersSubheaders" id="managers">Managers</h1>
-			<div id="managersList"></div>
+			<div id="managersList">Loding...</div>
```
Comment (Accio1) [inline forumhelpers/index.html:52]: Spelling error

**Review by Accio1 — CHANGES_REQUESTED** [review]
On the list of Forum Helpers page, the Loading text doesn't disappear after the list loads. This looks awkward after the list loads so it should disappear once the list is filled.

**Comment by leahcimto** [comment]
@Accio1 I was working on having it disappear, see a1cb746. I was having some trouble with that, does anyone have suggestions or see things that I'm doing wrong?

**Comment by Accio1** [comment]
> @Accio1 I was working on having it disappear, see [a1cb746](https://github.com/theforumhelpers/theforumhelpers.github.io/commit/a1cb7460750721c55c112818f5ee841a78122b35). I was having some trouble with that, does anyone have suggestions or see things that I'm doing wrong?

I went ahead and pushed a commit to fix it. You were using `.style.display = "hidden";` but it should be `.style.display = "none";`

I also condensed it down to one for loop instead of doing it separately for the curators and managers one. If you're good with these changes @leahcimto then we can go ahead and merge.

**Review by leahcimto — COMMENTED** [review]
Those changes look good. Thanks for your help!

**Comment by leahcimto** [comment]
Also to comply with the timeline in #325 do you mind disabling GH Pages in the QuickReply repository and then tomorrow archive it (or now for that matter)?

**Comment by Accio1** [comment]
> Also to comply with the timeline in #325 do you mind disabling GH Pages in the QuickReply repository and then tomorrow archive it (or now for that matter)?

Thanks for the reminder. I've disabled GitHub pages and I'll archive it tomorrow.

**Review by Accio1 — APPROVED** [review]

**Pull request merged**